components/camel-spring-parent/camel-spring-rabbitmq/src/test/java/org/apache/camel/component/springrabbit/integration/RabbitMQProducerIT.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.camel.RoutesBuilder;
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.springrabbit.SpringRabbitMQConstants;
+import org.apache.commons.lang3.time.StopWatch;
```

Comment:
We have our own `StopWatch` in `org.apache.camel.util`. I think it can do all of this for you.